modules/daggerverse/main.go

```diff
@@ -132,6 +132,10 @@ func (h *Daggerverse) BumpDaggerVersion(
 		from = strings.TrimSpace(from)
 	}
 
+	// get just the version, without the v semver prefix
+	from = strings.TrimPrefix(from, "v")
+	to = strings.TrimPrefix(to, "v")
+
```

Comment:
I did not expect this to be necessary. `awk -F'[v.]'` should have taken care of this. Will be taking a closer look at why this is an issue.

Reply:
I see! I was always dropping the `v` when specifying this value. It make sense that in CI the ref value has `v`, so this is a legit change.